README.md

```diff
@@ -27,37 +27,37 @@ These code samples are currently based on:
 
 ### Install the dependencies
 
-To install the dependencies for Android tests, run :
+To install the dependencies for JUnit4 tests, run :
 
-- For Junit4
+- For Android
 
     ```sh
-    cd android/junit4-examples
+    cd junit4-examples/android
```

Comment:
Let's make it consitent naming across aut and AA. junit-4/android and junit-5/android